src/components/probe/prober/http/request.ts

```diff
@@ -429,7 +437,7 @@ function getErrorStatusWithExplanation(error: unknown): {
         console.error(`Unhandled error, got ${(error as AxiosError).stack}`)
```

Comment:
pleaese add "error code 99" text also in the error log, since the alert sent is "code 99", but the logs don't have it. We need to connect logs to alerts when debugging.

Reply:
Consider it done